### PR TITLE
Adding in configuration snippets

### PR DESCRIPTION
**Pull request by WardenGnaw**
#1024

Creating snippets for local launch, local attach, local web launch, remote launch and remote attach.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1131,6 +1131,104 @@
...
+        "configurationSnippets": [
+            {
+                "label": ".NET: Launch .NET Core App",
+                "description": "Launch a .NET Core App with a debugger.",
+                "body": {
+                    "name": ".NET Core Launch (console)",
+                    "type": "coreclr",
+                    "request": "launch",
+                    "preLaunchTask": "build",
+                    "program": "^\"\\${workspaceRoot}/bin/Debug/<target-framework>/<project-name.dll>\"",
```
Comment (gregg-miskelly) [inline package.json]: Should we use their syntax to have the user fill in this path?

File: package.json [hunk 2]
```diff
@@ -1131,6 +1131,104 @@
             }
           }
         },
+        "configurationSnippets": [
+            {
+                "label": ".NET: Launch .NET Core App",
```
Comment (gregg-miskelly) [inline package.json]: Should we put 'Console' in the title?

File: package.json [hunk 3]
```diff
@@ -1131,6 +1131,104 @@
...
+                    "request": "launch",
+                    "preLaunchTask": "build",
+                    "program": "^\"\\${workspaceRoot}/bin/Debug/<target-framework>/<project-name.dll>\"",
+                    "args": [],
+                    "cwd": "^\"\\${workspaceRoot}\"",
+                    "stopAtEntry": false,
+                    "console": "internalConsole",
+                    "pipeTransport": {
+                        "pipeCwd": "^\"\\${workspaceRoot}\"",
+                        "pipeProgram": "enter the fully qualified path for the pipe program name, for example '/usr/bin/ssh'",
```
Comment (gregg-miskelly) [inline package.json]: 'pipeProgram' also?

File: package.json (hunk 1 above)
Comment (gregg-miskelly) [inline package.json]: Ex: 
    "program": "^\"\\${workspaceRoot}/bin/Debug/${1:<target-framework>/<project-name.dll>}\"",

**Comment by WardenGnaw** [comment]
@gregg-miskelly After looking deep into the vscode source.... You can add comments to the snippets. Instead of using the "body" keyword, use "bodyText". Should we change to that and include comments in our snippets?

**Comment by gregg-miskelly** [comment]
@WardenGnaw unless it is a huge amount of work, I think so.

**Comment by WardenGnaw** [comment]
bodyText will be used once snippets can call commands, so we can reuse the configurations in assets.

**Pull request merged**